dotcom-rendering/src/components/FlexibleGeneral.tsx

```diff
@@ -593,8 +597,16 @@ export const FlexibleGeneral = ({
 	collectionId,
 	isInHideTrailsAbTest,
 }: Props) => {
-	const splash = [...groupedTrails.splash].slice(0, 1);
-	const cards = [...groupedTrails.standard].slice(0, 19);
+	const splash = [...groupedTrails.splash].slice(0, 1).map((snap) => ({
```

Comment:
Given how we store cards in separate groupings (e.g. `splash`) and find them from `curated` and `backfill`, it was very fiddly to apply a unique index above this component.